erpnext_quota/install.py

```diff
@@ -36,7 +36,14 @@ def before_install():
         'used_company': 1,
         'count_website_users': 0,
         'count_administrator_user': 0,
-        'valid_till': add_days(today(), 14)
+        'valid_till': add_days(today(), 14),
+        'document_limit': [
+            {
+                'document_type': 'Sales Invoice',
+                'limit': 1,
```

Comment:
My suggestion is to set limit as 10 as 1 would be very low. The idea is that users can use the app out of the box
Add default limits for 

1. Sales Invoice
2. Purchase Invoice
3. Journal Entry
4. Payment Entry

Reply:
Done